subscription-service/src/main/resources/db/migration/VO_16_initialize_ geo_property.sql

```diff
@@ -0,0 +1,2 @@
+alter table geometry_query
+alter column geoproperty VARCHAR(64) NOT null default VARCHAR(64) 'https://uri.etsi.org/ngsi-ld/location'
```

Comment:
this does not seem like a valid request, how did you get it?

what's more, I'm not sure you can set a column as non null and set a default in a single command. so you should first set the default value then set as a not (`alter column geoproperty SET NOT null`)